src/Sylius/Bundle/CoreBundle/Fixture/Factory/PaymentMethodExampleFactory.php

```diff
@@ -132,6 +138,10 @@ private function getLocales()
     {
         /** @var LocaleInterface[] $locales */
         $locales = $this->localeRepository->findAll();
+        if (empty($locale)) {
```

Comment:
Can we move it above `$locales` declaration or below `foreach`? Right now it's messing with the `$locales` flow (we get the locales, **do something not related to them**, and then use them in foreach).